Cargo.toml

```diff
@@ -114,3 +116,13 @@ panic = "abort"
 [patch.crates-io.pq-sys]
 git = 'https://github.com/oxidecomputer/pq-sys'
 branch = "oxide/omicron"
+
+#
+# libsodium-sys is a dependency of thrussh, used in `end-to-end-tests`.  We
```

Comment:
We chatted about this off github, but for posterity's sake:

- We wanted libsodium because it's a dependency of thrussh
- We use thrussh in the end-to-end test
- Thrussh also looks deprecated

We should at least file an issue to follow-up here, and minimize our use of unsupported deps.